charts/sysdig/templates/configmap.yaml

```diff
@@ -46,6 +46,8 @@ data:
 {{- if $sslVerifyCertificate }}
     ssl_verify_certificate: {{ $sslVerifyCertificate }}
 {{- end }}
+    k8s_coldstart:
```

Comment:
let's add a check around this, so that this only gets added if someone hasn't defined `k8s_coldstart` in the `sysdig.settings`. while some yaml parsers will work around duplicate keys, it's really invalid

Reply:
good call, thanks!